code/modules/jobs/job_types/wasteland.dm

```diff
@@ -665,6 +666,21 @@ Raider
 							/obj/item/shovel=1,
 							/obj/item/gun/ballistic/automatic/pistol/m1911/compact=1)
 
+/datum/outfit/loadout/settler
+	name = "Settler"
+	uniform = /obj/item/clothing/under/f13/settler
+	shoes = /obj/item/clothing/shoes/f13/explorer
+    r_hand = /obj/item/hatchet
+    l_hand = /obj/item/gun/ballistic/automatic/pistol/n99
+    belt = /obj/item/storage/belt
+    backpack_contents = list(/obj/item/stack/sheet/metal=50,
+                            /obj/item/stack/sheet/mineral/wood=50,
+                            /obj/item/pickaxe/mini=1,
```

Comment:
```suggestion
                            /obj/item/pickaxe/min = 1,
```

Reply:
I'm pretty sure the correct object path is
/obj/item/pickaxe/mini

Based on 
code/modules/mining/equipment/mining_tools.dm